pyxrf/model/lineplot.py

```diff
@@ -454,26 +459,26 @@ def set_element(self, change):
         if '_K' in ename:
             e = Element(ename[:-2])
             if e.cs(incident_energy)['ka1'] != 0:
-                for i in range(4):
+                for i in range(k_len):
```

Comment:
can the `len` info be saved in the object `Element`? might be more sustainable

Reply:
good idea. but those are the same data structure for each element. So I just defined them in front.